app/lib/audit/models.dart

```diff
@@ -153,6 +153,30 @@ class AuditLogRecord extends db.ExpandoModel<String> {
       ..publishers = [];
   }
 
+  factory AuditLogRecord.packageVersionOptionsUpdated({
+    required String package,
+    required String version,
+    required User user,
+    required List<String> options,
+  }) {
+    final optionsStr = options.map((o) => '`$o`').join(', ');
+    return AuditLogRecord._init()
+      ..kind = AuditLogRecordKind.packageVersionOptionsUpdated
+      ..agent = user.userId
+      ..summary = '`${user.email}` updated $optionsStr of '
+          'package `$package` version `$version`.'
+      ..data = {
+        'package': package,
+        'version': version,
+        'user': user.email,
+        'options': options,
+      }
+      ..users = [user.userId]
+      ..packages = [package]
+      ..packageVersions = [version]
```

Comment:
The format of this field is unfortunately not documented, but it should be `'$package/$version'` as used in `packagePublished`. (The format is also used at different places e.g. `QualifiedVersionKey` class formalizes it).
Maybe add a note in the documentation of the field too?